src/rpc_connection.c

```diff
@@ -749,6 +748,7 @@ rpc_recv_msg(struct rpc_connection *conn, const void *frame, size_t len,
 	if (creds != NULL)
 		conn->rco_creds = *creds;
 
+	rpc_release(msg);
```

Comment:
Isn't that change going to bring back a memory leak when `msgt == NULL`?

Reply:
Good catch. I had changed that myself at one time and when merging picked the wrong placement for the release. Agreed that it should be released if the call to rpct_deserialize(msg) fails. I'll fix that.